frb/data/Galaxies/20180924B/FRB20180924B_host.json

```diff
@@ -141,21 +215,22 @@
         "WFC3_F300X_flux": 0.0016252695645897245,
         "WFC3_F300X_flux_err": 8.918272010276147e-05,
         "WFC3_F300X_ref": "Mannings2021",
-        "WISE_W1": 19.46128273010254,
-        "WISE_W1_err": 0.2150000035762787,
-        "WISE_W1_flux": 0.0596330501139164,
-        "WISE_W1_flux_err": 0.013059019111096859,
-        "WISE_W2": 19.291427612304688,
+        "WISE_W1": 19.461282732391798,
+        "WISE_W1_err": 0.2150000036,
+        "WISE_W1_flux": 0.05963303337719555,
+        "WISE_W1_flux_err": 0.013059014302971891,
+        "WISE_W1_ref": "Gordon2023",
+        "WISE_W2": 19.291362762451172,
         "WISE_W2_err": -999.0,
```

Comment:
if these are upper limits, they should be `999` and not `-999`

Reply:
Will rerun